### PR TITLE
Fix mobile topbar

### PR DESCRIPTION
**Pull request by marrero984**
screen shot 2017-02-27 at 11 19 23 am
screen shot 2017-02-27 at 11 19 21 am

### CODE REVIEW

File: public/js/showoff.js [hunk 1]
```diff
@@ -1689,7 +1689,7 @@ function setupStats(data)
 
 /* Is this a mobile device? */
 function mobile() {
-  return ( $(window).width() <= 480 )
+  return ( $(window).width() <= 640 )
```
Comment (binford2k) [inline public/js/showoff.js:1692]: what does 640 correspond to?

Comment (binford2k) [reply public/js/showoff.js:1692]: This is where the 480px came from originally, btw. https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/

Comment (marrero984) [reply public/js/showoff.js:1692]: The point at which nothing fits reasonably into the top bar.  That's what the media query came out to be, so I decided to update the function to match

**Comment by marrero984** [comment]
Fixed #690

**Pull request merged**